docs/conf.py

```diff
@@ -20,11 +20,9 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
 extensions = [
-    "autoapi.extension",
+    # "autoapi.extension",
```

Comment:
Curious, do you know why this was commented before?

Reply:
I think it may have been my fault - was playing around with different options for reference doc generation. We definitely don't need it.